fb_instant_articles.module

```diff
@@ -8,6 +8,38 @@
 use Drupal\Core\Routing\RouteMatchInterface;
 
 /**
+ * No ads will automatically be inserted into Instant Articles.
```

Comment:
@deviantintegral Is there a convention to follow for this, like class name, type, etc.?